### PR TITLE
Drush Cheat Sheet

### PR DESCRIPTION
**Pull request by jeet09**
**What does your Instant Answer do?**
- It's cheat sheet for drush command.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
- Show commands/cheat sheet for drush utility.

**What is the data source for your Instant Answer? (Provide a link if possible)**
- JSON

**Why did you choose this data source?**

**Are there any other alternative (better) data sources?**

**What are some example queries that trigger this Instant Answer?**
- drush commands or drush cheat sheet

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
- web developers, software engineers

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
- yes (https://duck.co/ia/view/drush_cheat_sheet)

**Which existing Instant Answers will this one supercede/overlap with?**
- None

**Are you having any problems? Do you need our help with anything?**
- Not yet

**Where did you hear about DuckDuckHack? (For first time contributors)**
- At my college time. Our college encourages Open Source Philosophy. [SICSR](http://sicsr.ac.in)

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
drush

## Checklist
Please place an 'X' where appropriate.

```
- [x] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [x] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [x] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [x] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
     - Ubuntu
        - [x] Google Chrome
        - [x] Firefox
        - [] Opera
      
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```
-----
IA Page: https://duck.co/ia/view/drush_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/drush.json [hunk 1]
```diff
@@ -0,0 +1,243 @@
+{
+    "id": "drush_cheat_sheet",
+    "name": "Drush",
+    "description": "shell scripting interface for Drupal",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/drush.json]: Let's capitalize "Shell"

File: share/goodie/cheat_sheets/drush.json [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    "metadata": {
+        "sourceName": "DrushCheatSheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/drush.json]: Please change to "Drush Commands"

**Comment by moollaza** [comment]
@jeet09 thanks a lot for contributing, we really appreciate it! There's just a couple tweaks to be made and this should be good to merge :+1:

**Comment by jeet09** [comment]
I'll modify it!

**Comment by mintsoft** [comment]
@jeet09 you can just add commits to the fork and they'll appear on the pr :)

**Comment by jeet09** [comment]
Made changes :)

**Comment by jeet09** [comment]
@moollaza Just out of curosity when it's going to be merged? :)

**Comment by mintsoft** [comment]
@jeet09 Apologies for the delay! If it looks ok to @moollaza it can be merged asap!

**Comment by jeet09** [comment]
@mintsoft @moollaza Still waiting! Sorry to say it again. :)

**Comment by moollaza** [comment]
LGTM :+1: 

@jeet09 sorry for the delay!

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Comment by jeet09** [comment]
@moollaza When I search "drush cheat sheet" I cannot see the results.

**Comment by mintsoft** [comment]
@jeet09 there are other steps for it to go through internally before it's live, someone will let you know once it's deployed :)

**Pull request merged**